scripts/test/lint.py

```diff
@@ -369,7 +370,7 @@ def check(self, path):
 INSTALL_LINTS = (
     MissesImportCommonVm(),
     FirstLineDoesNotSetErrorAction(),
-    UsesInvalidCategory(),
+    #UsesInvalidCategory(),
```

Comment:
As discussed in our last meeting, Please add a comment to explain why this is needed. I would also move to a separate commit than the one adding the `VM-Get-Category`, as the changes are completely independent.

Reply:
[nitpick] Still missing the comment here

Reply:
added comment now, I added it to a separate commit with the description so that's why I didn't add a comment